index.html

```diff
@@ -33,12 +33,12 @@
             <div class="collapse navbar-collapse" id="navbarCollapse">
                 <div class="navbar-nav">
                     <a href="index.html" class="nav-item nav-link active">Home</a>
-                    <a href="index2.html" class="nav-item nav-link active">Search for a book</a>
-                    <a href="index3.html" class="nav-item nav-link active">About us</a>
+                    <a href="/search book.html" class="nav-item nav-link active">Search for a book</a>
+                    <a href="/about us.html" class="nav-item nav-link active">About us</a>
```

Comment:
file names should be dashed not spaced between words

Reply:
I fixed it